Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html

```diff
@@ -59,7 +59,7 @@
               <td class="right" width='20%'>{{ id[0] }}</td>
               <td class="right" width='20%'>{{ id[1] }}</td>
               <td class="right" width='20%'>${{ id[2] }}</td>
-              <td><i class="fas fa-eye"></i></td>
+              <td (click)="openParticularOrder(id[0])"><i class="fas fa-eye"></i></td>
```

Comment:
why is this not running in a for loop?

Reply:
Id is running in for loop and we have to send the specific invoice number when  we click on it, to the update component

Reply:
This have been corrected